config.template.py

```diff
@@ -23,6 +21,7 @@
 
 # ARCSI CONF
 UPLOAD_FOLDER = "/abs/path/on/your/machine"  # this is where arcsi stores uploadable files temporarily
+ALLOWED_EXTENSIONS = ["mp3", "gif", "jpg", "jpeg", "png"]
```

Comment:
Isn't it strange to mix audio and image formats? Two lists (one for audio, one for image) would be more fault tolerant but I see that the current process() function doesn't know what it gets (image or audio), right?

Reply:
It is indeed strange and I first wrote `AUDIO` and `IMG` lists but as you say that was bit too ahead of its time. We don't yet know what it gets -- we will (have to) though, when we are at a point where we want to optimise processing based on what we are dealing with. 

But for now I went w/ the simplest working iteration